test/functional/buyers/service_contracts_controller_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class Buyers::ServiceContractsControllerTest < ActionController::TestCase
```

Comment:
These functional tests are deprecated and we intend to change them by integration tests 😄 So these tests should be in:
https://github.com/3scale/porta/blob/11e03b37712d0f73890eae5ddf5aca70bea10626/test/integration/buyers/service_contracts_controller_test.rb#L3

Reply:
good to know.. thanks 🍻